docs/extension-exceptions.md

```diff
@@ -0,0 +1,77 @@
+# Dealing with Browser-Extension Exceptions
+
+For most websites, the path for dealing with browser extension originating exceptions is simply to:
+
+1.  Identify the presence of problematic browser extensions
+2.  Disable Rollbar.js's reporting
+
+## Dealing with adblockers
+
+The most common type of extension that can be problematic is adblockers.  These extensions can disable loading
+certain external scripts or remove elements from a page based on a simple set of heuristics. 
+
+You can see a [full example](https://github.com/rollbar/rollbar.js/tree/master/examples/extension-exceptions/) 
+for details, but the approach can best summarized as below:
+
+Load Rollbar.js as normal.
+
+```html
+<script src="/assets/js/rollbar.js"></script>
+
+<script>
+  var _rollbarConfig = {
+    accessToken: 'POST_CLIENT_ITEM_ACCESS_TOKEN',
+    captureUncaught: true,
+    payload: {
+      environment: 'development',
+    }
+  };
+</script>
+```
+
+Add an html element with "bait" class names to be removed by adblockers.
+
+```html
+<!-- bait for adblocker like addons.  If they remove this div, we should disable error reporting -->
+<div id="blocker-bait" style="height: 1px; width: 1px; position: absolute; left: -999em; top: -999em"
+     class="ads ad adsbox doubleclick ad-placement carbon-ads"></div>
```

Comment:
AdBlock and AdGuard.  Though I suspect a larger amount of ad blockers would "fall" for the bait.  But there really isn't a perfect solution here -- ad blockers are in an arms race to circumvent "adblocker-blockers" and may change behavior in the future :/

But I really think this will work for the two most popular options I gave above.  Anything more becomes an exercise in patience and experimentation.